conductor-client/src/main/kotlin/com/openlattice/organizations/OrganizationEntitySetsService.kt

```diff
@@ -113,6 +137,34 @@ class OrganizationMetadataEntitySetsService(
             this::omAuthorizedPropertyTypes.isInitialized && this::datasetsAuthorizedPropertyTypes.isInitialized &&
             this::columnAuthorizedPropertyTypes.isInitialized && this::propertyTypes.isInitialized
 
+    fun initializeOrganizationMetadataEntitySets(organizationId: UUID) {
+        val adminAclKey = organizations.executeOnKey(
+                organizationId,
+                OrganizationReadEntryProcessor { it.adminRoleAclKey }
+        ) as AclKey
+
+        initializeOrganizationMetadataEntitySets(securePrincipalsManager.getRole(organizationId, adminAclKey[1]))
+    }
+
+    fun ensureOrganizationMetadataEntitySetIdsFullyInitialized(organizationId: UUID) {
+        val fullyInitialized = organizations
+                .executeOnKey(organizationId, OrganizationReadEntryProcessor { org ->
+                    with(org.organizationMetadataEntitySetIds) {
```

Comment:
this is some wild syntax lol -- what's happening here? Is this function being called on `org.organizationMetadataEntitySetIds`? I don't see where `org.organizationMetadataEntitySetIds` is referenced from inside the block, although I'm super not familiar with `with`